.luarc.json

```diff
@@ -9,7 +9,10 @@
       "${3rd}/busted/library"
     ],
     "ignoreDir": [
+      ".deps",
+      "build",
       "test",
+      "runtime/lua/uv/"
```

Comment:
Don't we need to **not** ignore this?

`runtime/lua` can be removed from `library` btw.

Reply:
We need to ignore this if we want don't want warnings in an upstream file to trip _our_ CI ;)

But I see you cleaned this up so we in fact don't need to ignore it. 👍 

(This was mostly continuing on from ignoring it for stylua, which we _do_ need to.)